src/Storages/StorageMerge.h

```diff
@@ -177,31 +177,39 @@ class ReadFromMerge final : public SourceStepWithFilter
 
     using Aliases = std::vector<AliasData>;
 
+    class RowPolicyData;
+
     static SelectQueryInfo getModifiedQueryInfo(const SelectQueryInfo & query_info,
         const ContextPtr & modified_context,
         const StorageWithLockAndName & storage_with_lock_and_name,
         const StorageSnapshotPtr & storage_snapshot);
 
+    void processAliases(
+        Names & real_column_names,
+        const StorageWithLockAndName & storage_with_lock,
+        Aliases & aliases,
+        const Block & sample_block,
+        ContextMutablePtr modified_context);
```

Comment:
What exactly the function `processAliases()` does? I know it's named `processAliases` but how it processes them? A comment here would be useful.

Reply:
I've moved this part of code as is, but Ok, I'll try to add some comments.